### PR TITLE
The Great Resprite: Atomized edition (Stock Parts volume)

### PR DESCRIPTION
**Pull request by SomeAngryMiner**
## About The Pull Request

You remember the big PR i did that resprited everything on engineering? i got asked to atomize it into something that aint an OmniPR, so here we go.
Original PR: https://github.com/BeeStation/BeeStation-Hornet/pull/2339/
## Why It's Good For The Game

Visual distinction and uniqueness! if the entire set gets merged, we might get an entire, unified pallette.

image

## Changelog
:cl:
imgadd: Resprites the stock parts to a new, fresh style,
/:cl:

### CODE REVIEW

**Comment by Cenrus** [comment]
Very good, I also love the lil smiley face

**Comment by Blueturbo47** [comment]
Cool but if your ever make a new t1 make t1 new t2 and make a tier for t2 which is the parts but red new t2.

**Comment by SomeAngryMiner** [comment]
> Cool but if your ever make a new t1 make t1 new t2 and make a tier for t2 which is the parts but red new t2.

wh
What?????

**Comment by ike709** [comment]
Overall good but I feel like the perspective is weird on the new power cells.

**Comment by FishFireTBoy** [comment]
try to cut back on the banding a bit imo, much better than the old ones tho

**Comment by Isy232** [comment]
+9000, I like all the new ones very much

**Comment by LordSaladin** [comment]
Nice

File: code/game/machinery/cell_charger.dm [hunk 1]
```diff
@@ -16,7 +16,6 @@
 /obj/machinery/cell_charger/update_icon()
 	cut_overlays()
 	if(charging)
-		add_overlay(image(charging.icon, charging.icon_state))
```
Comment (qwertyquerty) [inline code/game/machinery/cell_charger.dm:19]: Why is this removed? You seem to have edited the overlays for it?

Comment (SomeAngryMiner) [reply code/game/machinery/cell_charger.dm:19]: i edited the charger sprite, that adds the powercell as an overlay, wich wouldnt show through the lid.

**Review by qwertyquerty — APPROVED** [review]
